.github/workflows/t3000-model-perf-tests.yaml

```diff
@@ -11,7 +11,13 @@ jobs:
     with:
       arch: '["wormhole_b0"]'
     secrets: inherit
+  build-artifact-profiler:
+    uses: ./.github/workflows/build-artifact.yaml
+    with:
+      arch: '["wormhole_b0"]'
+      tracy: true
+    secrets: inherit
   t3000-model-perf-tests:
-    needs: build-artifact
+    needs: [build-artifact, build-artifact-profiler]
```

Comment:
Can we just use only the build-artifact-profiler here?
I don't think we need build-artifact anymore if we want to use profiler tools here.

Reply:
I have introduced an if statement in the impl file to choose which build to use for that job